CHANGELOG.md

```diff
@@ -68,6 +74,16 @@ and this project adheres to
   cause an increase in the memory footprint of the Firecracker process. Now, if
   we receive such a frame, we ignore it and increase `Net::tx_malformed_frames`
   metric.
+- [#4536](https://github.com/firecracker-microvm/firecracker/pull/4536): Make
+  the first differential snapshots taken after a full snapshots contain only the
+  set of memory pages changed since the full snapshot. Previously, these
+  differential snapshots would contain all memory pages.
+- [#4578](https://github.com/firecracker-microvm/firecracker/pull/4578): Fix
+  UFFD support not being forward-compatible with new ioctl options introduces in
+  newer kernel versions. See also
+  https://github.com/bytecodealliance/userfaultfd-rs/issues/61.
+- [#4567](https://github.com/firecracker-microvm/firecracker/pull/4567): Do not
+  initialize VCPUs in powered-off state upon snapshot restore.
```

Comment:
We can add a sentence explaining that we only need to power off them on boot, but not in restore.

nit: vCPU is more common in our doc than VCPU.

Reply:
done (and done)